### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by ramakrishnach14**
Cards feature implemented. Cards with text only , with badges , with dismiss , with text overlay , vertical card , card with shadow.

### CODE REVIEW

File: components/cards.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    border-radius: 2%;
+    position: relative;
+
+}
+
+.card-header{
+    font-size: 1.5rem;
+    font-weight: 600;
+    padding-left: 1rem;
+    padding-top: 1rem;
```
Comment (MaruthiPothuganti) [inline components/cards.css:14]: This can be done by using short-hand property as  _padding: 1rem 0rem 0rem 1rem;_

File: components/cards.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.card-badge{
+    width: 10em;
+    height:3em;
+    text-align: center;
+    padding: 1rem;
+    border-radius: 4%;
+    background-color: chartreuse;
```
Comment (omgaikwad) [inline components/cards.css:38]: you can use color variables here

File: components/cards.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+.bottom-right{
+    bottom: 1rem;
+    right:1rem;
+}
+
+.card-vertical{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
```
Comment (MaruthiPothuganti) [inline components/cards.css:97]: This can be made into a utility class and can be reused

**Review by omgaikwad — APPROVED** [review]
Other than the requested changes code looks good!

File: index.html [hunk 4]
```diff
@@ -57,6 +57,72 @@ <h4>Badge on Avatars</h4>
...
+    <div class="card">
+        <img src="https://placehold.co/200x200" alt="image" class="card-image-full">
+        <div class="card-overlay top-left">Top-left</div>
+        <div class="card-overlay top-right">Top-right</div>
+        <div class="card-overlay bottom-left">bottom-left</div>
+        <div class="card-overlay bottom-right">bottom-right</div>
+    </div>
+    <h4>Vertical Card</h4>
+    <div class="card card-vertical">
+        <img src="https://placehold.co/200x200"/>
```
Comment (MaruthiPothuganti) [inline index.html:112]: Alt attribute can be added for accessibility

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of changes. Everything else looks good

**Pull request merged**